docs/introduction.mdx

```diff
@@ -84,9 +84,13 @@ const sum = atom((get) => atoms.map(get).reduce((acc, count) => acc + count))
 ```
 
 ### Derived async atoms
-<img src="https://img.shields.io/badge/-needs_suspense-black" alt="needs suspense" />
 
-You can make the read function an async function too.
+<img
+  src="https://img.shields.io/badge/-needs_suspense-black"
+  alt="needs suspense"
+/>
```

Comment:
The image tag must be in one line to render properly.

Reply:
Whoops! I think that must have been prettier. Thanks. I've fixed it :smile: